logger.go

```diff
@@ -121,6 +129,21 @@ func (l *Logger) output(s severity, txt string) {
 	}
 }
 
+// Close closes all the underlying log destinations, which will flush all the
+// cached logs. Please call Close once all logging is done.
```

Comment:
Drop the second and third sentence:

Close closes all the underlying log writers, which will flush any cached logs.
Any errors from closing the underlying log writers will be printed to stderr.
Once Close is called, all future calls to the logger will panic.

Reply:
Done.